part-configuration.tex

```diff
@@ -187,20 +187,245 @@ \section{Matching your business processes}
+
+The main setup consists of the following:
+
+\begin{description}[style=nextline]
+    \item[Account Number] Income statements and balance sheets reports are ordered by account number. Traditionally, account numbers are 4 digits with accounts being grouped by the first digit, but this type of grouping is not enforced by LedgerSMB. Grouping is handled using headings as explained in \secref{sec-coa-accounts-structure}.
+    \item[Description] Should be a relative short, easy to understand description of what the account is used for.
+    \item[Account Type] Asset, Liability, Equity, Equity temporary, Income or Expense.
+    \item[Heading] Optional entry. Used if this account is a child of a heading account.
+    \item[Negative balance heading] @@@ TODO
+\end{description}
+
+\subsubsection{Account configuration}
+\label{subsec-coa-account-configuration}
+
+The items in the configuration box as shown in \figref{fig:coa-account-setup} are used to configure the account for use in LedgerSMB.
+
+There are currently three types of optional summary accounts:
+
+\begin{description}[style=nextline]
+    \item [AR] Marking an account as a summary account for AR means that all outstanding
+    receivable amounts will be posted to this account. The Accounts Receivable administration
+    will contain the details of which amount is owed by which \gls{customer}.
+    \item [AP] Same as the AR account, except for amounts owed to vendors.
+    \item [Inventory] This account holds the monetary value equal to the items on stock.
+\end{description}
+
+Each account may have zero, one or more of the following options selected.
+
+\begin{description}[style=nextline]
+    \item[Contra] This checkmark identifies the account as a \gls{contra}\index{contra}account, which means
+    that the account is going to hold the opposite of an account it's associated with.
+    A good example of this kind would be the depreciation account associated with a fixed
+    asset account where the depreciation account contains the credit amount to be added to
+    the original asset (debit) value to get the current asset value.
+    \item[Recon] This checkmark identifies the account as one which needs reconciliation\index{reconciliation} as
+    described in \secref{sec-business-processes-accounting-reconciliation}.
+    \item[Tax] This checkmark identifies the account as a Tax (VAT)\index{tax} account. Tax accounts need
+    to be further configured. See \charef{cha-taxes} for further discussion of the
+    subject.
+    \item[Obsolete] This checkmark identifies the account as not being usable going forward.  The account cannot be deleted because it may
+    contain data.
+    \label{item:AccountOptionsTax}
+\end{description}
+
+
+\subsubsection{Include in drop-down menus}
+\label{subsec-coa-account-links}
+
+In order to enhance the user interface accounts may be added to drop down menus throughout LedgerSMB.

```

Comment:
But note that this does not apply to accounts which have been marked as "Summary" accounts.

Reply:
Added clarity and expanded paragraph.